openmls/src/group/core_group/test_past_secrets.rs

```diff
@@ -16,7 +16,7 @@ fn test_secret_tree_store(ciphersuite: &'static Ciphersuite, backend: &impl Open
 
     // Make sure we can access the message secrets we just stored
     assert!(message_secrets_store
-        .secrets_for_epoch(GroupEpoch(0))
+        .secrets_for_epoch_mut(GroupEpoch(0))
```

Comment:
Doesn't `secrets_for_epoch` work here? All these calls don't need mutability.

Reply:
`secrets_for_epoch` was unused and I deleted it.